nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -344,6 +344,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - It is now possible to specify wordlists to include as handy to access environment variables using the `config.environment.wordlist` configuration options.
 
+- GNOME now defaults to using pipewire only, whereas before the default configuration would lead to both pulseaudio and pipewire being enabled. This should fix some issues that arose due to the two being enabled before.
```

Comment:
The issues have been resolved so I would write something like:

> GNOME module now enables PipeWire as the audio server instead of PulseAudio. Shims for ALSA and PulseAudio for apps that do not yet support PipeWire natively are enabled as well. This should, among other things, improve out-of-the-box support for Bluetooth audio devices.